grammars/repositories/flavors/math-block.cson

```diff
@@ -1,13 +1,35 @@
 key: 'math-block'
 patterns: [
+  {
+    name: 'block.math.markup.md'
+    begin: '(\\${2})'
+    end: '(\\${2})(?:.*)'
+    patterns: [{ include: 'text.tex.latex' }]
+    beginCaptures:
+      1: name: 'punctuation.md'
+    endCaptures:
+      1: name: 'punctuation.md'
+  }
 
-  name: 'block.math.markup.md'
-  begin: '(\\${2})'
-  end: '(\\${2})(?:.*)'
-  patterns: [{ include: 'text.tex.latex' }]
-  beginCaptures:
-    1: name: 'punctuation.md'
-  endCaptures:
-    1: name: 'punctuation.md'
+  {
+    name: 'block.math.markup.md'
+    begin: '\\\\\\['
+    end: '\\\\\\]'
+    patterns: [{ include: 'text.tex.latex' }]
+    beginCaptures:
+      1: name: 'punctuation.md'
+    endCaptures:
+      1: name: 'punctuation.md'
+  }
```

Comment:
This is the correct number of backslashes, four `\` to represent a single literal `\`, and then two `\` to escape the `[`.

If it's not showing up correctly, it's the fault of language-latex and their use of `$base`. See https://github.com/burodepeper/language-markdown/pull/226, https://github.com/area/language-latex/pull/185.

On my branch of language-latex with the changes mentioned in https://github.com/area/language-latex/pull/185, I see:

![image](https://user-images.githubusercontent.com/15164633/42195958-71d721c0-7e49-11e8-9146-4f013eeba752.png)